.github/config/android-arm.json

```diff
@@ -4,20 +4,22 @@
         "android-arm"
     ],
     "targets": [
+        "ipc_tests",
+        "content_shell",
+        "system_webview_apk",
+        "system_webview_shell_apk",
+        "cobalt:gn_all"
+    ],
+    "test_targets": [
```

Comment:
Change to `unit_test_targets`?

Not super important for this temporary solution but I think it's good practice anyhow.

Reply:
done